foreman_puppet.gemspec

```diff
@@ -13,4 +13,6 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] + ['LICENSE', 'Rakefile', 'README.md', 'package.json']
   s.test_files = Dir['test/**/*']
+
+  s.required_ruby_version = '>= 2.7', '< 4'
```

Comment:
Why `< 4`? It should be just `~> 2.7.0`

Reply:
`~> 2.7.0` would prevent it from running on Ruby 3, which is something we'd like to do.